cmd/updater/main.go

```diff
@@ -136,9 +137,11 @@ func main() {
 	groupUpdater := updater.GCS(opt.groupTimeout, opt.buildTimeout, opt.buildConcurrency, opt.confirm, updater.SortStarted)
 	updateOnce := func() {
 		start := time.Now()
+		cycle := metrics.NewLogInt64("cycle_duration", "How long an update cycle took, in seconds.", logrus.New())
```

Comment:
Shouldn't we just create this once, before updateOnce definition? (so L137 or so?)

Reply:
Yup, thanks for catching! 

Also manually verified that this will successfully log when the Updater is run.